README.md

```diff
@@ -31,7 +31,7 @@ rustup target add wasm32-unknown-unknown --toolchain nightly
 
 Once this is configured you can build and test the repo as follows:
 
-```bash
+```
```

Comment:
This one is actually bash:)

Reply:
Yeah, but the highlighting is weird

<img width="510" alt="image" src="https://user-images.githubusercontent.com/12344623/121204741-1b39bf00-c845-11eb-9335-c4d83018f559.png">

Looks better without